app/src/main/java/com/rajatgoyal/puzzle15/ui/GameActivity.java

```diff
@@ -76,28 +75,22 @@ protected void onCreate(Bundle savedInstanceState) {
         Timber.d("onCreate: ");
 
         Intent intent = getIntent();
+        boolean resumeGame = false;
```

Comment:
Better to keep `resumeGame` as a field in GameActivity, it may be accessed from some other Activity, in fact, the check for `resumeGame` should be in the MainActivity itself, and probably we can show a resume button for that case, views @rajatgoyal715 ? Send the param inside param to `GameActivity` instead

Reply:
yeah it can be made a field if it's used anywhere else in the GameActivity. As for checking in MainActivity, this variable is already coming from there in intent.

Reply:
Oh my bad, I just saw that it's correct in that case.

@rajatgoyal715 you can add a resume button probably in the `MainActivity` in case `resumeGame` is true which in turn will load the intended layout with the desired **state**